python/FlexSource.py

```diff
@@ -85,6 +85,23 @@ def set_bandwidth(self,bandwidth):
         self._bandwidth = self.__hz_to_mhz(bandwidth)
         self.pan_adapter.Bandwidth = self._bandwidth
 
+    @property
+    def rx_ant(self):
+        """
+        Returns RX antenna in use.
+        """
+        return self._rx_ant
+
+    def set_rx_ant(self, rx_ant):
+        """
+        Sets the RX antenna of the underlying Panadapter
+
+        Args:
+            rx_ant: the new RX antenna to use
+        """
+        self._rx_ant = self.rx_ant
+        self.pan_adapter.RXAnt = self._rx_ant
```

Comment:
Very nice, on the docstring comments.  Did someone get pylint up and running?

Reply:
No idea abut pylint haha. Just trying to stick with conventions.